bundles/org.openhab.ui/web/src/components/widgets/system/oh-repeater.vue

```diff
@@ -22,15 +22,6 @@ export default {
     Fragment
   },
   widget: OhRepeaterDefinition,
-  data () {
-    return {
-      ready: false,
```

Comment:
You need to add `sourceCache` to data, otherwise Vue will complain during runtime.